docs/Home.md

```diff
@@ -1,58 +1,72 @@
-Welcome to the Terasology Wiki!
+<div class="img-responsive">
+  <img src="/images/terasology-logo.png" alt="Terasology Banner" width="300">
+</div>
 
-This is a wiki for those interested in contributing to the development of the project.
 
-It doesn't cover game content or how to play the game much, but check out the docs on [Playing](https://github.com/MovingBlocks/Terasology/blob/develop/docs/Playing.md) (including hotkeys etc) and [Modules](https://github.com/MovingBlocks/Terasology/blob/develop/docs/Modules.md) for some of that.
+# Welcome to Terasology!
```

Comment:
I feel like this "home" page should be the split-off point to the "hubs" for the different audiences (players, devs, maintainers), similar to how we did it in http://terasology.org/Terasology/#/Troubleshooting

Also, I feel like this page is very very long - ideally I would imagine it as the jump-off point into the dedicated hubs + some references to the most important links to get additional help (community/support), which btw I think are duplicated in "Community and Support", "Getting Help", "Troubleshooting", and "Stay Updated & Reach Out"

Reply:
You are right about making the homepage split-off point to the "hubs" for different audiences. What about the quick links section? It can cover different audience, and regarding the duplicates, we can have "Getting Help and Support," which refer to opening an issue in GitHub, and "Troubleshooting," which refer to the troubleshooting guide. The Announcement Channels remain the same.

Reply:
we can keep some quick links on the "home" page, yes, but we should consider what would be relevant for all audiences to get direct access to. I think troubleshooting, opening an issue, and maybe a link to our discord server might be reasonable candidates. any further ideas?